packages/digit_components/lib/widgets/atoms/digit_reactive_dropdown.dart

```diff
@@ -22,6 +23,7 @@ class DigitReactiveDropdown<T> extends StatelessWidget {
     required this.formControlName,
     this.isRequired = false,
     this.isDisabled = false,
+    this.readOnly = false,
     required this.valueMapper,
```

Comment:
@amitverma-egov can we directly use isDisabled??

Reply:
@vishal-egov sure will check